entgql/extension.go

```diff
@@ -201,6 +209,14 @@ func WithSchemaGenerator() ExtensionOption {
 	}
 }
 
+// WithCustomScalars adds custom scalar types to the schema.
+func WithCustomScalars(scalars ...CustomScalar) ExtensionOption {
```

Comment:
Why do you need this? This information can be extracted from the GoType field, by checking that it implements the GQL marshaller interfaces.

Reply:
Are all types implements the GQL marshaller interfaces is a scalar? How we can check for UUID case? It's not implement GQL marshaller interfaces, but using custom marshal functions
```
  UUID:
    model:
      - entgo.io/contrib/entgql/internal/todouuid/ent/schema/uuidgql.UUID
```
 This allow user defined their custom scalar type with more information

Reply:
Got it, agree.